modules/rosa-basic-concepts.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// understanding-rosa/understanding-rosa.adoc
+
+
+[id="rosa-basic-concepts"]
+= Basic Concepts
+
+This topic explains some basic concepts about the container platform.
+
+[id="rosa-kubernetes_{context}"]
+== Kubernetes
+
+{product-title} (ROSA) uses Red Hat’s enterprise Kubernetes platform. Kubernetes is an open source platform for managing containerized workloads and services across multiple hosts, and offers management tools for deploying, automating, monitoring, and scaling containerized apps with minimal to no manual intervention. For complete information about Kubernetes, see the link:https://kubernetes.io/docs/home/?path=users&persona=app-developer&level=foundational[Kubernetes documentation].
+
+Account:: Your account refers to your AWS account.
```

Comment:
"Account" seems a little out of place here, given it's for an AWS account, but I get that it's a bit orphaned. Maybe better under the "OpenShift" concept, especially if it ends up moving above the "Kubernetes" concept (per other comment)?

Reply:
Well, since there are several accounts used at different times, I removed this reference. We specify which account to use in the appropriate procedure. I think that will be more clear ultimately.